src/main/java/karuna/karuna_backend/DTO/UserDTO.java

```diff
@@ -9,5 +9,7 @@
 @Builder
 public class UserDTO {
     private String username;
+
+    //TODO: remove this field
```

Comment:
U have right, but it should be in diffrent pull requsts, one case one pull request should be